### ISSUE TITLE
Add - Today button

### ISSUE THREAD

**Issue by S1M8N** [nielsfaber/scheduler-card#589]
### Checklist

- [X] I checked for similar existing requests (both open and closed) before posting.
- [X] My request is generic, other users may benefit from it too.
- [X] I will participate in further discussion about this feature and can test it (if requested) once it's done.

### Proposal

Hello everyone working on this project,

To begin, thank you for taking the time for this great tool.

Since the time that I use the planner, there is one thing on which I had to restrict myself, it is the planning of an action on a single day.

The purpose would be to plan an action "Today at xx hours" and simply "Today", once the action passes, the task will turn off on its own.

This button could be put the "DAYS" category in the "CHOOSE" button or directly next to "WEEKEND".

![2022-08-15 13_14_15-Overview – Home Assistant - Opera](https://user-images.githubusercontent.com/75216525/184625955-0d5b5dc3-4133-4cbf-8a4d-43b9fb4a1925.png)

### Additional info

Personally I will use this feature to plan an automation or other during the day. Once the action passes, the planning is deactivated and during a new desire, I can activate it again.

I remain available for everything.

Thank you very much for your help to the HA community

Simon

**Comment by nielsfaber** [comment]
The field you are referring to is for the days of the week. Today, Tomorrow, etc. are not part of that since they are relative to the current date (i.e. they would only be valid once, afterwards your schedule would be corrupted).

If you want to have a schedule trigger only once, you should use the 'behaviour after triggering' under tab 'options' to select whether to repeat, disable, or delete after the schedule was triggered.
![image](https://user-images.githubusercontent.com/16861605/185777332-d28062b2-a596-4881-a41e-d91ba4b7e7a1.png)

In other words, what you're looking for is already in the card but in a different way. You can leave the days (of the week) on the default setting (daily) and set 'behaviour after triggering' as desired.
